src/goto-programs/goto_convert_class.h

```diff
@@ -236,7 +236,7 @@ class goto_convertt:public messaget
     goto_programt &dest,
     const irep_idt &mode);
   void convert_cpp_delete(const codet &code, goto_programt &dest);
-  void convert_loop_invariant(
+  void convert_loop_contract(
```

Comment:
```suggestion
  void convert_loop_contracts(
```

(here and elsewhere)